polyfill/lib/calendar.mjs

```diff
@@ -1098,8 +1100,8 @@ const helperHebrew = ObjectAssign({}, nonIsoHelperBase, {
   },
   adjustCalendarDate(calendarDate, cache, overflow = 'constrain', fromLegacyDate = false) {
     let { year, eraYear, month, monthCode, day, monthExtra } = calendarDate;
-    if (year === undefined) year = eraYear;
-    if (eraYear === undefined) eraYear = year;
+    if (year === undefined && eraYear !== undefined) year = eraYear;
+    if (eraYear === undefined && year !== undefined) eraYear = year;
```

Comment:
These are also optional (required in TS to avoid assigning a may-be-undefined `eraYear` to a never-undefined `year`) but IMHO add clarity here too.  LMK if you'd like me to revert.

Reply:
removed this change; not important enough to port.